src/distributed/mpi.c

```diff
@@ -191,9 +191,6 @@ void mpi_remote_msg_handle(void)
 				continue;
 			}
 			msg = msg_allocator_alloc(0);
-			// make sure the deterministic tie-breaking doesn't read uninitialized data
-			msg->m_type = 0;
-			msg->pl_size = 0;
```

Comment:
This one I am not sure why.

Reply:
We don't need to write default values inside received anti-messages anymore. 
Now, we don't use the extended tiebreak inside the queue, but rather we rely on the straggler detection to handle them. So remote anti-messages will never have their members inspected (because the processing logic will immediately recognise them as anti-message and take proper action) and we can afford to not initialise them.

Reply:
It is clear. I'm always reluctant to leave values uninitialised explicitly, especially when it costs so little to initialise them, because you never know if future changes might enter undefined behaviour because of a commit years earlier. Let's move on for now, but try to avoid such things in general.